SemiBin/main.py

```diff
@@ -312,7 +312,7 @@ def generate_cov_multiple(bam_file, bam_index, out, threshold,
     bam_name = os.path.split(bam_file)[-1] + '_{}'.format(bam_index)
     bam_depth = os.path.join(out, '{}_depth.txt'.format(bam_name))
     os.system(
```

Comment:
This would still not work if the path has `'` characters or something.

Please just use `subprocess.check_call` instead of `os.system` to make all of these issues go away. Also, it adds error checking.

Reply:
OK. I will fix it.